File: ios/Classes/entity/FindGroupApplicationEntity.swift
//
// Created by 蒋具宏 on 2020/10/28.
//

import Foundation

/// 查找群申请信息实体
class FindGroupApplicationEntity: NSObject {

    /// 来自用户
    var fromUser: String?;

    /// 群ID
    var groupID: String?;

    required public override init() {
    }

    convenience init(json: String) {
        self.init(dict: JsonUtil.getDictionaryFromJSONString(jsonString: json))
    }

    init(dict: [String: Any]) {
        super.init();
        self.fromUser = (dict["fromUser"] as! String);
        self.groupID = (dict["groupID"] as! String);
    }
}

File: lib/entity/find_group_application_entity.dart
/// 查找群申请实体
class FindGroupApplicationEntity {
  /// 来自用户
  String fromUser;

  /// 群ID
  String groupID;

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data["fromUser"] = fromUser;
    data["groupID"] = groupID;
    return data;
  }
}
